app/src/comic/models.py

```
@@ -89,6 +87,28 @@ class Meta:
+            ),
+            GinIndex(
+                fields=["author_tw"],
+                name="series_author_tw_trgm_idx",
+                opclasses=["gin_trgm_ops"],
+            ),
+            GinIndex(
+                fields=["author_jp"],
+                name="series_author_jp_trgm_idx",
+                opclasses=["gin_trgm_ops"],
+            ),
+        ]
```

Ruff (0.14.10)

Mutable class attributes should be annotated with `typing.ClassVar`

(RUF012)